game_catalog/serializers.py

```diff
@@ -0,0 +1,83 @@
+from rest_framework import serializers
+from .models import Genre, Studio, Game, Comment, CustomUser
+
+
+class RegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=8)
+    email = serializers.EmailField(required=False, allow_blank=True, default="")
+    first_name = serializers.CharField(required=False, allow_blank=True, default="")
+    last_name = serializers.CharField(required=False, allow_blank=True, default="")
+
+    class Meta:
+        model = CustomUser
+        fields = ["username", "first_name", "last_name", "email", "password"]
```

Comment:
According to Django best practices if we use `ModelSerializer `and we already provided field in `Meta `class it will reuse ORM `Model `logic, so we don't need to explicitly mention them in serializer fields.